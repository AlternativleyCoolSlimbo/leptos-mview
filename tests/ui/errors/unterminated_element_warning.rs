use leptos_mview::view;

fn main() {
    _ = view! {
        div {
            "something"
            input.input type="text"
        }
    };
    compile_error!("test warnings");
}
