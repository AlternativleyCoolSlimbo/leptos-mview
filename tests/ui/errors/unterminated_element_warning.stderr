warning: unterminated element
 --> tests/ui/errors/unterminated_element_warning.rs:7:13
  |
7 |             input.input type="text"
  |             ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: elements without a `;` or children block is only allowed for better rust-analyzer support. do not leave elements unterminated to avoid ambiguities

error: test warnings
  --> tests/ui/errors/unterminated_element_warning.rs:10:5
   |
10 |     compile_error!("test warnings");
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
